### ISSUE TITLE
Feature suggestion : Favorites/saves messages

### ISSUE THREAD

**Issue by Dryhb** [LAION-AI/Open-Assistant#1348]
As mentionned in https://github.com/LAION-AI/Open-Assistant/issues/1331 , some messages are really good / funny / original and as the author suggested a display for everyone on the main page I think an option to save/place as favorite/follow a message to be able to see it (and the children/ancestors of it) on a special page (maybe on the profile under a new page like followed trees / messages ?).
Sometimes you see a good question and are simply interested by the answer or the initial prompt is a funny one... Maybe even be a kind of leaderboard, who gets the most follow on a message ...

**Comment by mm12** [comment]
I agree. When I respond to a prompt or make one, I want to see the outcome.
